packages/wonder-blocks-core/components/clickable-behavior.test.js

```diff
@@ -337,9 +343,11 @@ describe("ClickableBehavior", () => {
         button.simulate("keyup", {keyCode: keyCodes.enter});
         expect(onClick).toHaveBeenCalledTimes(3);
 
-        button.simulate("touchstart", {keyCode: keyCodes.space});
-        button.simulate("touchend", {keyCode: keyCodes.space});
-        button.simulate("click");
```

Comment:
I'm curious as to why you removed this click? Wouldn't we expect this to still fire?

Reply:
nope, because `preventDefault()` on touchend means that none of the `mouse` events fire.